README.md

```diff
@@ -60,7 +60,20 @@ dism { 'Internet-Explorer-Optional-amd64':
 ~~~
 
 ##Reference
-
+###Deployment on Azure Windows Server 2012 R2 Datacentre (Image from Azure Store)
+####Windows Server 2012 R2 Datacentre does not have dism installed under c:\windows\sysnative 
+####it is under c:\windows\system32
```

Comment:
Also, if you take a look here at https://github.com/puppetlabs/puppetlabs-dism/blob/master/lib/puppet/provider/dism/dism.rb#L7-L14, you will notice we search for the command in several places. 

What version of the DISM module do you have?